### PR TITLE
added #merge! to Readme

### PR DESCRIPTION
**Pull request by ASnow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,19 @@ end
 # => "author": { "name": "David" }
 ```
 
+
+To merge existing hash or array to current context:
+
+``` ruby
+hash = {author: {name: "David"}}
+json.post do
+  json.tilte "Merge HOWTO"
```
Comment (dirk) [inline README.md]: Spelling error.

File: README.md [hunk 2]
```diff
@@ -77,6 +77,19 @@ end
...
+  json.merge! hash
+end
+
+# => "post": {title: "Merge HOWTO", "author": { "name": "David" }}
+```
+
```
Comment (jeremy) [inline README.md]: Spaces around `{ … }` to match the code style in the rest of the README.

**Comment by ASnow** [comment]
@dirk, @jeremy changed

**Comment by rwz** [comment]
Could you add quotes around `title`? Also, could you squash your commits into one?

**Comment by prathamesh-sonpatki** [comment]
This looks good apart from one comment above. @ASnow Can you take care of that?

**Comment by ASnow** [comment]
Ok. Will do soon.

**Comment by pixeltrix** [comment]
Edited and merged manually in 3b3fc6d17ae346bece1a433ccafa8828abd6af15

**Pull request closed without merging**